output-tests/content/draft-next/general.json

```diff
@@ -0,0 +1,43 @@
+[
+    {
+        "description": "failed validation produces no annotations",
+        "schema": {
+            "$schema": "https://json-schema.org/draft/next/schema",
+            "$id": "https://json-schema.org/tests/content/draft-next/general/1",
+            "type": "string",
+            "readOnly": true
+        },
+        "tests": [
+            {
+                "description": "dropped annotations MAY appear in droppedAnnotations",
+                "data": 1,
+                "output": {
+                    "basic": {
+                        "type": "object",
+                        "properties": {
+                            "details": {
+                                "contains": {
+                                    "type": "object",
+                                    "properties": {
+                                        "evaluationPath": {"const": ""},
+                                        "schemaLocation": {"const": "https://json-schema.org/tests/content/draft-next/general/1"},
+                                        "instanceLocation": {"const": ""},
+                                        "annotations": false,
+                                        "droppedAnnotations": {
+                                            "properties": {
+                                                "readOnly": {"const": true}
+                                            },
+                                            "required": ["readOnly"]
+                                        }
+                                    },
+                                    "required": ["evaluationPath", "schemaLocation", "instanceLocation"]
+                                }
+                            }
+                        },
+                        "required": ["details"]
+                    }
+                }
+            }
+        ]
+    }
+]
```

Comment:
Can you configure your editor to not strip the newline at the end?

Reply:
Might be available in .editorconfig if you don't mind having that.

Reply:
Yeah definitely fine with adding one of those.

Reply:
✔️